docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: '3'
```

Comment:
Do we need this file?

Reply:
No. I had this for generating the documentation locally but this can be removed.